packages/wallet/src/services/TransactionsTracker.ts

```diff
@@ -277,6 +281,18 @@ export const createTransactionsTracker = (
     share()
   );
 
+  const newSignedOrPreviouslySigned$ = merge<SignedTx[]>(
```

Comment:
This observable looks almost exactly the same as `submittingOrPreviouslySubmitted$`. Would be great to extract a util function to keep it dry

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/1113/commits/656aead1c7734afcf0375489c1c5a82080527545